packages/grafana-data/src/datetime/formatter.ts

```diff
@@ -5,28 +5,89 @@ import { DateTimeInput } from './moment_wrapper';
 import { DEFAULT_DATE_TIME_FORMAT, MS_DATE_TIME_FORMAT } from './formats';
 import { DateTimeOptions, getTimeZone } from './common';
 
+/**
+ * Type describing the formatting options that can be passed to the {@link dateTimeFormat}
```

Comment:
Pretty sure this "type" is a noun, but I recommend making this parallel to the "type" in the earlier example when you decide how to clarify it.